pynicotine/nowplaying.py

```diff
@@ -259,6 +261,41 @@ def mpris(self, player):
 
         return True
 
+    def listenbrainz(self, username):
+        """ Function to get the currently playing song listenbrainz api """
+
+        import json
+
+        if not username:
+            log.add_important_error(_('listenbrainz: Please provide your listenbrainz username'))
+            return
+
+        try:
+            response = http_request('https', 'api.listenbrainz.org', f'/1/user/{username}/playing-now',
```

Comment:
It doesn't seem like a good idea to have URL's hard-coded into the core plugin handler, especially if third-party plugin developers need to do this. We should consider a better way of hooking data into the required entries from the /plugins folder.

Reply:
Remove this f-string for Python 3.5 compatibility.